docs/OnTrack/Deployment/Enhanced Authentication/current-and-proposed-authentication-evaluation-5.1.md

```diff
@@ -0,0 +1,185 @@
+  OnTrack, this follows recommended password procedures which suggests that passwords should not be
+  reused across various accounts.
+- Additionally, this extension of the user password management would allow users to request a
+  password change from the administrator in the event that they have forgotten their password and
+  cannot gain access to their account. This is a useful feature for account management and access,
+  ensuring that users are able to regain access to their account swiftly and easily.
+- The addition of extra functions allowing user management to be achieved will improve upon the
+  current authentication setup through allowing more user qualities to be managed independently by
+  the admins and, for some features, by the users themselves. Overall, this adds additional
+  flexibility and ease of change for certain parameters or information.
+- Regarding the LDAP server utilising Devise, this will allow the OnTrack system to have its own
+  in-house authentication mechanisms with complete control over how it is set up, integrated, and
+  deployed. As the current authentication mechanism relies on third-party “Single Sign On” (SSO)
+  authentication protocols provided by Deakin University, OnTrack is limited to utilising the
+  mechanisms provided by this third-party system. Developing an in-house authentication solution
+  using LDAP and Devise results in the OnTrack architecture being fully independent, and allows for
+  additional sign-in options to be added in future (for example, using a Google Account to sign in).
+- Additionally, the use of the LDAP Devise server option allows for the authentication systems to be
+  fully managed by the OnTrack team, allowing for complete control over the code and features
+  included as part of the authentication processes. This is an improvement as reliance on
+  third-party software or processes in any circumstance increase the difficulty in understanding how
+  the systems operate and what has occurred in the instance of an error or breach. Having an
+  independent authentication solution streamlines the control and medication of the services
+  provided by it, allowing for access and control to be granted solely to the OnTrack team on the
+  backend of the project. In this way, the LDAP and Devise features used can be customised to suit
+  the specific (and changing) needs of the OnTrack system, using only features that are necessary
+  and relevant to the OnTrack system and user authentication needs, rather than accepting all
+  services provided by third-party authentication mechanisms
+- As stated, the OnTrack team will be granted full control over the authentication systems and
+  elements of the OnTrack software. This will solve the issue of the redirection to the broken
+  Deakin SSO link as explored in previous sections, as the team will be able to control the flow
+  towards other pages after the successful signing out of a user.
```

Comment:
Suggestion: I would consider a table? Might help with readability.
Example below: You can use the text you have but format it differently

|Element |Advancement |
|---|---|
|Password management| Forgot password|
|Devise LDAP | access control|
| | |